setup/azurecmdfiles.wxi

```diff
@@ -462,9 +462,6 @@
                         <Component Id="cmp6BDCB170E1F314882CF58F8F80DE76A6" Guid="*">
                             <File Id="filBFDE1699047559AB8DA89E0F660D35DF" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.Billing\Microsoft.Azure.Commands.Common.Authentication.dll" />
                         </Component>
-                        <Component Id="cmpED253190CB7DAF0A995567435152DC6A" Guid="*">
```

Comment:
@markcowl all dll-Help.xml references are still in the wxi file - the ones being removed are from projects with dependencies on other projects (_e.g.,_ Profile, Tags, Resources)